Spotify UI/lib/widgets/horizontal_scroll_list.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:insta/helpers/size.dart';
+
+class HorizontalScrollList extends StatelessWidget {
+  HorizontalScrollList(this.albumList);
+
+  final dynamic albumList;
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      child: Row(
+        children: [
+          for (var album in albumList)
+            Container(
+              padding: EdgeInsets.only(
+                left: getWidthOfDevice(context) * 0.03,
+                right: getWidthOfDevice(context) * 0.04,
```

Comment:
Not sure why the padding needs to be based on the width of the device and why left and right are different, no big deal anyway